front-end/cypress/support/transaction_nav_trees.spec.ts

```diff
@@ -18,14 +18,16 @@ export type SchATransactionName =
   | 'Party Receipt'
   | 'PAC Receipt'
   | 'Transfer'
-  | 'Earmark Receipt';
+  | 'Earmark Receipt'
+  | 'Joint Fundraising Transfer - National Party Recount Account';
```

Comment:
The transaction names that are used throughout this file are associated with the transaction accordion on the front-end.  Those entries are sometimes different from their Unique Transaction Identifiers.